cosmovisor/README.md

```diff
@@ -37,22 +37,21 @@ To install a previous version, you can specify the version. IMPORTANT: Chains th
 go install github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor@v0.1.0
```

Comment:
can we update the version here, or put <select_version or latest. people tend to copy fast